src/OpenFeature.Contrib.Providers.Flagd/FlagdProvider.cs

```diff
@@ -387,16 +390,40 @@ private static Value ConvertToPrimitiveValue(ProtoValue value)
         private static Service.ServiceClient buildClientForPlatform(Uri url)
         {
             var useUnixSocket = url.ToString().StartsWith("unix://");
+            X509Certificate2 certificate = null;
 
             if (!useUnixSocket)
             {
+                var flagdCertPath = Environment.GetEnvironmentVariable("FLAGD_SERVER_CERT_PATH") ?? "";
```

Comment:
You may want to sync up with @bacherfl - I think we should also have a constructor that takes a config object, see [here](https://github.com/open-feature/dotnet-sdk-contrib/pull/47#discussion_r1152321671).

Reply:
Agree, I think Flo's PR should be merged first and afterwards I can rebase and adapt the changes